08_analysis_tools/CalibrateHallSensor.m

```diff
@@ -86,6 +97,17 @@
 % TODO: this could be calculated from the filter gain itself...
 cal_opt.idx_ed_crop = 3;
 
+% ----------------
+% Options for detection mode 1
```

Comment:
Would maybe be nice to put a short description of what each detection mode does -- otherwise.. I'm also not opposed to simple removing the old method (but still a description would be good).

Reply:
Added